libfreerdp/core/peer.c

```diff
@@ -267,6 +267,7 @@ static BOOL freerdp_peer_initialize(freerdp_peer* client)
 		}
 	}
 
+	nego_set_RCG_supported(rdp->context->rdp->nego, settings->RemoteCredentialGuard);
```

Comment:
looks a little recursive? `rdp->nego` should be sufficient, should it not?

Reply:
good catch